test/e2e/egressip.go

```diff
@@ -761,7 +761,7 @@ spec:
 		if utilnet.IsIPv6String(egress2Node.nodeIP) {
 			otherDstIP = "fc00:f853:ccd:e793:ffff::1"
 		} else {
-			otherDstIP = "172.18.1.1"
+			otherDstIP = "172.18.1.99"
```

Comment:
haha, which other test is using this?
btw I am using 172.18.1.3 or something for another EFW test.
the key is to remove and cleanup in the aftereach of every test so that this cross over doesn't happen.
are we giving the same IP out to two nodes in the same test?

Reply:
`Should validate the egress SVC SNAT functionality against host-networked pods`

Reply:
>are we giving the same IP out to two nodes in the same test?

No. 2 tests I can see are using the same IP but not at the same time. Tests are cleaning up the IP after themselves - removing that IP.

Reply:
hmm so I am confused, if at a given time only 1test is using the given IP and after that it is getting cleanedup before it is applied to the next node, is it the fact that the mac binding entry's timeout has not reached so it still has the mac of the oldnode?

Reply:
yes

Reply:
ack I am lgtm,
can you please update the PR description or commit description by elaboraing more on:
```
Two e2es shared the node IP 172.18.1.1 and assigned them to different nodes. This caused some e2es to timeout due to stale mac binding entry.
``` 
so that anyone passing by this PR can get the whole picture/
once you do that I will merge this, sorry for being a pusher here

Reply:
self-note: takes 5mins for the entry to expire

Reply:
@martinkennelly also said this is just short-term fix to get around the CI flake. We should have a better plan to not allocate sameIPs which have a MAC binding already? Even that I guess is a hack, Ideally once the IP get's deleted we should not be having issues for like 5mins from being able to reuse this :/
We should track this work somewhere

Reply:
Done